pages/filterCards/index.tsx

```diff
@@ -0,0 +1,64 @@
+import axios from "axios";
+// tslint:disable no-submodule-imports
+import { withRouter } from "next/router";
+import * as React from "react";
+
+interface IProps {
```

Comment:
TS recomend not to use `I` (at least for TS development)

Reply:
I've read suggestions when I run `yarn lint` and it was like a warning - that's why I've added `I`